js/views/invite_contacts_dialog_view.js

```diff
@@ -15,7 +15,12 @@
       const convos = window.getConversations().models;
 
       this.contacts = convos.filter(
-        d => !!d && !d.isBlocked() && d.isPrivate() && !d.isMe()
+        d =>
+          !!d &&
+          !d.isBlocked() &&
+          d.isPrivate() &&
+          !d.isMe() &&
+          !!d.attributes.timestamp
```

Comment:
do we want to make a function in `conversations.js` such as `isActive()`? This might make it easier to read in places where we use `d.attributes.timestamp`?